pkgs/by-name/pg/pg-gvm/package.nix

```
@@ -0,0 +1,60 @@
+{
+  cmake,
+  fetchFromGitHub,
+  glib,
+  gvm-libs,
+  icu,
+  lib,
+  libical,
+  pcre2,
+  pkg-config,
+  postgresql,
+  stdenv,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pg-gvm";
+  version = "22.6.5";
+
+  src = fetchFromGitHub {
+    owner = "greenbone";
+    repo = "pg-gvm";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-19ZmQdLjfwJwOMoO16rKJYKOnRyt7SQOdkYTxt8WQ2A=";
+  };
+
+  strictDeps = true;
+
+  configurePhase = ''
+  runHook preConfigure
+
+    cmake \
+      -DCMAKE_INSTALL_DEV_PREFIX=$out .
+
+  runHook postConfigure
```

```suggestion
    runHook postConfigure
```
Indentation is wrong.